### PR TITLE
Added VPC Accessible Services configuration

### PR DESCRIPTION
**Pull request by masterleros**
Adding missing configuration for VPC Accessible Services

### CODE REVIEW

File: modules/regular_service_perimeter/variables.tf [hunk 1]
```diff
@@ -117,3 +117,15 @@ variable "ingress_policies_dry_run" {
   }))
   default = []
 }
+
+variable "vpc_accessible_services" {
+  description = "A list of [VPC Accessible Services](https://cloud.google.com/vpc-service-controls/docs/vpc-accessible-services) that will be restricted within the VPC Network. Use [\"RESTRICTED-SERVICES\"] to match the restricted services list."
```
Comment (bharathkkb) [inline modules/regular_service_perimeter/variables.tf:122]: We should also call out that `*`  is a special value we defined that disables vpc_accessible_services and allows all services.

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @masterleros 
Could you also sign the CLA

**Comment by masterleros** [comment]
> Thanks for the PR @masterleros Could you also sign the CLA

@bharathkkb CLA is already signed, all the changed are done, not sure why the build has failed, could you please retrigger it?
Thank you

File: modules/regular_service_perimeter/variables.tf [hunk 2]
```diff
@@ -117,3 +117,15 @@ variable "ingress_policies_dry_run" {
   }))
   default = []
 }
+
+variable "vpc_accessible_services" {
+  description = "A list of [VPC Accessible Services](https://cloud.google.com/vpc-service-controls/docs/vpc-accessible-services) that will be restricted within the VPC Network. Use [\"*\"] to allow any service (disable VPC Accessible Services); Use [\"RESTRICTED-SERVICES\"] to match the restricted services list; Use [] to do not allow any service."
```
Comment (bharathkkb) [inline modules/regular_service_perimeter/variables.tf:122]:
```suggestion
  description = "A list of [VPC Accessible Services](https://cloud.google.com/vpc-service-controls/docs/vpc-accessible-services) that will be restricted within the VPC Network. Use [\"*\"] to allow any service (disable VPC Accessible Services); Use [\"RESTRICTED-SERVICES\"] to match the restricted services list; Use [] to not allow any service."
```

File: modules/regular_service_perimeter/variables.tf [hunk 3]
```diff
@@ -117,3 +117,15 @@ variable "ingress_policies_dry_run" {
...
+  type        = list(string)
+  default     = ["*"]
+}
+
+variable "vpc_accessible_services_dry_run" {
+  description = "(Dry-run) A list of [VPC Accessible Services](https://cloud.google.com/vpc-service-controls/docs/vpc-accessible-services) that will be restricted within the VPC Network. Use [\"*\"] to allow any service (disable VPC Accessible Services); Use [\"RESTRICTED-SERVICES\"] to match the restricted services list; Use [] to do not allow any service."
```
Comment (bharathkkb) [inline modules/regular_service_perimeter/variables.tf:128]:
```suggestion
  description = "(Dry-run) A list of [VPC Accessible Services](https://cloud.google.com/vpc-service-controls/docs/vpc-accessible-services) that will be restricted within the VPC Network. Use [\"*\"] to allow any service (disable VPC Accessible Services); Use [\"RESTRICTED-SERVICES\"] to match the restricted services list; Use [] to not allow any service."
```

**Review by bharathkkb — APPROVED** [review]
minor nit

**Pull request merged**